modules/util/source-formatter/src/main/resources/documentation/check/model_set_call_with_company_id_check.md

```diff
@@ -0,0 +1,21 @@
+## ModelSetCallWithCompanyIdCheck
+
+Do not use variable `companyId` as the parameter when calling model set method,
+unless set method explicitly references companyId, like `setCompanId`.
+
+### Examples
+
+Incorrect:
+
```

Comment:
@marianoalvarosaiz
SF can only catch the simply cases like: 
```
model.setDescription (“Any description “ + companyId); or
model.setDescription (companyId);
```

So please update the document for one of those cases, and the correct changes in `Correct:` section for it, thanks.

Reply:
@ling-alan-huang I've just updated it.